driver-dist/pom.xml

```diff
@@ -70,44 +70,6 @@
                         </goals>
                         <configuration>
                             <includeDependencySources>true</includeDependencySources>
-                            <!-- optional dependencies from other modules (must be explicitly declared here in order to be correctly resolved) -->
-                            <additionalDependencies>
-                                <additionalDependency>
-                                    <groupId>org.xerial.snappy</groupId>
-                                    <artifactId>snappy-java</artifactId>
-                                    <version>${snappy.version}</version>
-                                </additionalDependency>
-                                <additionalDependency>
-                                    <groupId>net.jpountz.lz4</groupId>
-                                    <artifactId>lz4</artifactId>
-                                    <version>${lz4.version}</version>
-                                </additionalDependency>
-                                <additionalDependency>
-                                    <groupId>org.hdrhistogram</groupId>
-                                    <artifactId>HdrHistogram</artifactId>
-                                    <version>${hdr.version}</version>
-                                </additionalDependency>
-                                <additionalDependency>
-                                    <groupId>com.fasterxml.jackson.core</groupId>
-                                    <artifactId>jackson-core</artifactId>
-                                    <version>${jackson.version}</version>
-                                </additionalDependency>
-                                <additionalDependency>
-                                    <groupId>com.fasterxml.jackson.core</groupId>
-                                    <artifactId>jackson-databind</artifactId>
-                                    <version>${jackson.version}</version>
-                                </additionalDependency>
-                                <additionalDependency>
-                                    <groupId>joda-time</groupId>
-                                    <artifactId>joda-time</artifactId>
-                                    <version>${joda.version}</version>
-                                </additionalDependency>
-                                <additionalDependency>
-                                    <groupId>javax.json</groupId>
-                                    <artifactId>javax.json-api</artifactId>
-                                    <version>${jsr353-api.version}</version>
-                                </additionalDependency>
-                            </additionalDependencies>
```

Comment:
For some reason this change never made it to the 3.1.x branch, will fix when we merge downstream.